### PR TITLE
security-monitoring: add a small guide on how to setup security filters via the API

### PR DESCRIPTION
**Pull request by jbillon**
### What does this PR do?
add a small guide on how to setup security filters via the API

### Motivation
add a few examples to explain a bit how to use the security filters

### Preview

https://docs-staging.datadoghq.com/julien.billon/security-filters-api-guide/security_platform/guide/how-to-setup-security-filters-using-security-monitoring-api/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/guide/how-to-setup-security-filters-using-security-monitoring-api.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+kind: guide
+aliases:
+  - /security_monitoring/guide/how-to-setup-security-filters-using-security-monitoring-api/
+---
+
+## Overview
+
+The Security Monitoring product analyzes your ingested logs to detect threats in real time, such as by matching logs with threat intelligence, or by applying [Security Rules][1] to detect attacks or anomalies.
+
+Datadog charges for analyzed logs based on the total number of gigabytes ingested and analyzed by the Datadog Security Monitoring service. By default, Security Monitoring analyzes all your ingested logs to maximize detection coverage. However, by using the [Security Monitoring API][2], you can programmatically setup "Security Filters" to configure which subset of ingested logs to analyze.
```
Comment (sarina-dd) [inline content/en/security_platform/guide/how-to-setup-security-filters-using-security-monitoring-api.md:12]:
```suggestion
Datadog charges for analyzed logs based on the total number of gigabytes ingested and analyzed by the Datadog Security Monitoring service. By default, Security Monitoring analyzes all your ingested logs to maximize detection coverage. However, by using the [Security Monitoring API][2], you can programmatically setup Security Filters to configure which subset of ingested logs to analyze.
```

File: content/en/security_platform/guide/how-to-setup-security-filters-using-security-monitoring-api.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+
+* An API key and an application key **from an admin user** is required to use the API. These are available in your [Datadog account API key page][3]. Replace `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your Datadog API key and your Datadog application key.
+
+* This guide features `curl` examples. Install [curl][4] if you do not have it installed, or reference additional language examples for this API endpoint in the [API documentation][2].
+
+## Examples
+
+### Add an exclusion to the default Security Filter
+
+By default, a single Security Filter exists that analyzes all ingested logs. It's named "all ingested logs" and has a query of `*`. You can customize it by adding an exclusion to exclude a subset of logs based on their tags. To do so, you first need to retrieve the list of Security Filters in order to get the filter's `id`.
```
Comment (sarina-dd) [inline content/en/security_platform/guide/how-to-setup-security-filters-using-security-monitoring-api.md:29]:
```suggestion
By default, a single Security Filter exists that analyzes all ingested logs. It's named `all ingested logs` and has a query of `*`. You can customize it by adding an exclusion to exclude a subset of logs based on their tags. To do so, you first need to retrieve the list of Security Filters in order to get the filter's `id`.
```

File: content/en/security_platform/guide/how-to-setup-security-filters-using-security-monitoring-api.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+            "id": "l6l-rmx-mqx"
+        }
+    ]
+}
+
+```
+
+In this example, the filter's `id` is `"l6l-rmx-mqx"`. You can then modify it to add an exclusion, for example exclude all the logs tagged with `env:staging`.
+
+Note: `version` indicates the current version of the filter you want to update.
```
Comment (sarina-dd) [inline content/en/security_platform/guide/how-to-setup-security-filters-using-security-monitoring-api.md:68]:
```suggestion
**Note**: `version` indicates the current version of the filter you want to update.
```

File: content/en/security_platform/guide/how-to-setup-security-filters-using-security-monitoring-api.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+        "type": "security_filters",
+        "id": "l6l-rmx-mqx"
+    }
+}
+
+```
+
+### Create a custom Security Filter
+
+You can also create custom Security Filters in order to restrict the analysis to explicitly specified logs. For example you can choose to analyze the logs from AWS Cloudtrail with a filter that matches only `source:cloudtrail`.
```
Comment (sarina-dd) [inline content/en/security_platform/guide/how-to-setup-security-filters-using-security-monitoring-api.md:122]:
```suggestion
You can also create custom Security Filters in order to restrict analysis to explicitly specified logs. For example, you can choose to analyze logs from AWS Cloudtrail with a filter that matches only `source:cloudtrail`.
```

**Review by pbruneton-dd — APPROVED** [review]

File: content/en/security_platform/guide/how-to-setup-security-filters-using-security-monitoring-api.md [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+| AWS Network Logs      | `source:(vpc OR waf OR elb OR alb)`              |
+| GCP Logs              | `source:gcp*`                                    |
+| Azure Logs            | `source:azure*`                                  |
+| Kubernetes Audit Logs | `source:kubernetes.audit`                        |
+| Identity Provider Logs| `source:(okta OR gsuite OR auth0)`               |
+| CDN Logs              | `source:(cloudfront OR cloudflare OR fastly)`    |
+| Web Server Logs       | `source:(nginx* OR apache OR iis)`               |
+
+[1]: /security_platform/detection_rules/security_monitoring
+[2]: /api/v2/security-monitoring/
```
[the remark this answers is not in the document]
Comment (sarina-dd) [reply content/en/security_platform/guide/how-to-setup-security-filters-using-security-monitoring-api.md:246]: Approved the API spec, will put a DNM label on this until Monday and we can update the link once live 👍

**Pull request merged**